src/modules.py

```diff
@@ -196,13 +198,10 @@ def run(self) -> None:
         hpr = self.helper
 
         while True:
-            vers = hpr.get_vers()
-            if not vers:
-                break
+            popped = hpr.pop_from_queue()
```

Comment:
Can you use a separate change/review/patchset/commit for the pop_from_queue concurrency issue, and just make this review about the browser window size issue?

Reply:
I see! I will separate `pop_from_queue` concurrency issue and `ensuring browser` issue!